addon/ng2/utilities/ts-path-mappings-webpack-plugin.ts

```diff
@@ -1,252 +0,0 @@
-import * as path from 'path';
```

Comment:
It was the result of us waiting for a PR to be accepted on ATL. PR is in, and I had to update the ATL version, so I removed it.